htdocs/PI/write/PIWriteRequest.php

```diff
@@ -799,6 +816,17 @@ private function updateService(Site $siteService){
         $this->updateServiceExtensionProperties($service);
         break;
       }
+      case 'endpoint':{
+        if ($this->requestMethod != 'DELETE') {
+          $this->exceptionWithResponseCode(400,
+            "Only deletion of endpoints is supported when a service is specified. ".
```

Comment:
Should this be the following? It's not clear to me...
```suggestion
            "Deletion of endpoints is only supported when a service is specified. ".
```

Reply:
I believe this has been superseded by a refactor - I can't find it in the current codebase